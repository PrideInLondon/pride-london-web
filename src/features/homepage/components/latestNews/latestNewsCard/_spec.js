import React from 'react'
import { shallow } from 'enzyme'
import LatestNewsCard from '.'

describe(LatestNewsCard.name, () => {
  it('renders with default values', () => {
    const title = 'Here is some news!'
    const newsCategory = {
      hexColour: '#304242',
      title: 'Amazing News!!',
    }
    const readLength = ''
    const date = '2019-03-28T20:30+00:00'
    const images = [{ file: { url: 'backgroundimageurl/image.png' } }]
    const wrapper = shallow(
      <LatestNewsCard
        newsCategory={newsCategory}
        title={title}
        date={date}
        readLength={readLength}
        images={images}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
