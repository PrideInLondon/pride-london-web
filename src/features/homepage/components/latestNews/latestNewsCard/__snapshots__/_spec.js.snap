// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestNewsCard renders with default values 1`] = `
<styles__Card>
  <styles__CardHeader>
    <styles__NewsCategory
      backgroundColor="#304242"
    >
      Amazing News!!
    </styles__NewsCategory>
    <styles__NewsMetrics>
      <NewsDate
        date="2019-03-28T20:30+00:00"
      />
    </styles__NewsMetrics>
  </styles__CardHeader>
  <styles__CardTitle>
    <styles__UnderlineContainer>
      Here is some news!
    </styles__UnderlineContainer>
  </styles__CardTitle>
</styles__Card>
`;
